lib/Workflow/Action.pm

```diff
@@ -198,16 +199,15 @@ This documentation describes version 1.56 of this package
      # Since 'username' and 'email' have already been validated we
      # don't need to check them for uniqueness, well-formedness, etc.
 
-     my $user = eval {
-         User->create({ username => $context->param( 'username' ),
-                        email    => $context->param( 'email' ) })
-     };
-
-     # Wrap all errors returned...
-
-     if ( $@ ) {
+     my $user;
+     try {
+         $user = User->create({ username => $context->param( 'username' ),
+                                email    => $context->param( 'email' ) })
+     }
+     catch ($err) {
```

Comment:
Can we rename this to `$error` over `$err` - I prefer long form

Reply:
Yes. (just mentioning: it's a documentation/code example, not actual code.)